npm/react/cypress/plugins/index.js

```diff
@@ -60,7 +60,11 @@ const webpackConfig = {
 /**
  * @type Cypress.PluginConfig
  */
-module.exports = (on, config) => {
+module.exports = (on, config, mode) => {
+  if (mode !== 'component') {
+    throw Error('This is an component project. mode should be `component`.')
```

Comment:
This is a good way to test that `mode` is passed correctly. We expect this project to be only used with runner-ct.